prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,445 @@
+          "shield_advanced_protection_in_global_accelerators",
+          "shield_advanced_protection_in_internet_facing_load_balancers",
+          "shield_advanced_protection_in_route53_hosted_zones",
+          "ecr_repositories_scan_images_on_push_enabled",
+          "ecr_repositories_scan_vulnerabilities_in_latest_image"
+        ]
+      },
+      {
+        "Id": "Budget-Alarms",
+        "Name": "Budget-Alarms",
+        "Description": "Budget Alarms for your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Prod",
+            "Name": "Prod"
+          },
+          {
+            "Section": "QA",
+            "Name": "QA"
+          }
+        ],
+        "Checks": []
+      },
+      {
+        "Id": "IAM-Access-Analyzer",
+        "Name": "IAM access analyzer",
+        "Description": "IAM access analyzer in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Analyzer-Regions",
+            "Name": "Create analyzers in each active regions"
+          },
+          {
+            "Section": "Verify-that-events-are-present-in-SecurityHub-aggregated-view",
+            "Name": "Verify that events are present in SecurityHub aggregated view"
+          }
+        ],
+        "Checks": [
+          "securityhub_enabled"
+        ]
+      },
+      {
+        "Id": "Macie",
+        "Name": "Macie",
+        "Description": "Macie in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Critical-Buckets",
+            "Name": "Consider enabling for critical buckets only"
+          }
+        ],
+        "Checks": [
+          "macie_is_enabled"
+        ]
+      },
+      {
+        "Id": "Vulnerability-Scanning",
+        "Name": "Vulnerability Scanning",
+        "Description": "Vulnearbility Scanning in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "EC2-used-as-servers",
+            "Name": "Apply-existing-SPCS-based-on-OU-placement"
+          },
+          {
+            "Section": "ECR-used-as-docker-image-hub",
+            "Name": "ECR used as docker image hub"
+          }
+        ],
+        "Checks": [
+          "inspector2_findings_exist",
+          "ecr_repositories_scan_images_on_push_enabled",
+          "ecr_repositories_lifecycle_policy_enabled",
+          "ecr_repositories_scan_vulnerabilities_in_latest_image"
+        ]
+      },
+      {
+        "Id": "AWS-Clousdtrail",
+        "Name": "AWS Cloustrail",
+        "Description": "Cloudtrail in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Organization-Trail",
+            "Name": "Enable as part of Organization trail"
+          },
+          {
+            "Section": "Confirm-Logs-S3-SIEM",
+            "Name": "Confirm that logs are present in S3 bucket and SIEM"
+          }
+        ],
+        "Checks": [
+          "cloudtrail_cloudwatch_logging_enabled",
```

Comment:
Can you add these ones?
<img width="397" alt="image" src="https://github.com/prowler-cloud/prowler/assets/38561120/9e5609df-f1fa-4f7e-9bd7-128f2b68d31f">

Reply:
Done!